src/test/resources/alma-fix/991005935279706485.json

```diff
@@ -80,6 +80,105 @@
     "label" : "Zeitschrift",
     "id" : "http://d-nb.info/gnd/4067488-5"
   } ],
+  "subject" : [ {
+    "notation" : "650",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "RPB (Rheinland-Pfälzische Bibliographie)",
+      "id" : "http://bartoc.org/en/node/1990"
+    }
+  }, {
+    "notation" : "120",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "RPB (Rheinland-Pfälzische Bibliographie)",
+      "id" : "http://bartoc.org/en/node/1990"
+    }
+  }, {
+    "notation" : "ZS 222",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "RVK (Regensburger Verbundklassifikation)",
+      "id" : "https://d-nb.info/gnd/4449787-8"
+    }
+  }, {
+    "notation" : "ZS 0001",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "RVK (Regensburger Verbundklassifikation)",
+      "id" : "https://d-nb.info/gnd/4449787-8"
+    }
+  }, {
+    "notation" : "22",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "DNB (Systematik der Deutschen Nationalbibliographie)",
+      "id" : "http://bartoc.org/en/node/18497"
+    }
+  }, {
+    "notation" : "17",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "DNB (Systematik der Deutschen Nationalbibliographie)",
+      "id" : "http://bartoc.org/en/node/18497"
+    }
+  }, {
+    "notation" : "370",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "DNB (Systematik der Deutschen Nationalbibliographie)",
+      "id" : "http://bartoc.org/en/node/18497"
+    }
+  }, {
+    "notation" : "330",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "DNB (Systematik der Deutschen Nationalbibliographie)",
+      "id" : "http://bartoc.org/en/node/18497"
+    }
+  }, {
+    "notation" : "3,2",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "ssgn",
+      "id" : "ssgn"
+    }
+  }, {
+    "notation" : "5,3",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "ssgn",
+      "id" : "ssgn"
+    }
+  }, {
```

Comment:
This seems to be an odd result.

Reply:
found this: https://wiki.obvsg.at/Katalogisierungshandbuch/Kategorienuebersicht084SE

084 mit $$2 ssgn oder $$2 fid (Fremddaten)

> In den Fremddaten findet man häufig die Kategorie 084 mit diesen beiden Unterfeldern.
> ssgn = Sondersammelgebietsnummer
> fid = Fachinformationsdienste für die Wissenschaft
> 
> Was sind Sondersammelgebiete?
> Um sicherzustellen, dass jedes wissenschaftlich relevante Buch in mindestens einem Exemplar in Deutschland vorhanden ist, wurde bis 2015 die kooperative Erwerbung der gesamten wissenschaftlichen Literaturproduktion von der Deutschen Forschungsgemeinschaft (DFG) organisiert. Dafür wurden Bibliotheken ausgewählt, die für bestimmte Fächer bereits umfassende Spezialbestände besaßen. 36 Bibliotheken sammelten 110 Sondersammelgebiete. Jedes Wissensgebiet war einer bestimmten Bibliothek in Deutschland zugeteilt; diese sammelte so umfassend und vollständig wie möglich die weltweit erscheinende Literatur dazu.
> Jedem Fachgebiet wurde eine Nummer zugewiesen. Jene deutschen Bibliotheken, die Medien im Rahmen der DFG-Sondersammelgebietsförderung erschlossen haben, erfassten diese Nummer in 084 $$a mit $$2 ssgn. (→ [Übersicht der SSG-Nummern](https://wikis.sub.uni-hamburg.de/webis/index.php?title=SSG-Verteilungsplan&oldid=9832))
> Das Nachfolgeprogramm der SSG nennt sich Fachinformationsdienste für die Wissenschaft (FID). (→ [Übersicht der Fachinformationsdienste](https://sigel.staatsbibliothek-berlin.de/fid-kennzeichen/))

I just could update the element, but these are not really subjects but collections.
I would suggest to kick those notations out!

Reply:
> I would suggest to kick those notations out!

+1